.storybook/main.ts

```diff
@@ -14,6 +14,9 @@ export const framework: StorybookConfig['framework'] = '@storybook/react';
 
 export const staticDirs: StorybookConfig['staticDirs'] = ['./public'];
 
+const path = require('path');
+const globalCSS = ['./src/assets/styles/main.scss'];
```

Comment:
Does this path get rewritten in the bundle so that GH Pages will work?

Reply:
Not quite sure yet but working on storybook, might need your help on this one as well

Reply:
It looks like webpack bundles the CSS into the JS (or it just gets completely compiled away), because with `npm run storybook:preview`, using the built output, I don't see any CSS requests. So this concern should be a non-issue.